backend/apps/github/index/issue.py

```
@@ -78,11 +78,21 @@
 
     @staticmethod
     def update_synonyms():
-        """Update synonyms."""
+        """
+        Update synonyms for the issue index.
+
+        Returns:
+            None
+        """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)